torch/csrc/api/include/torch/nn/modules/functional.h

```diff
@@ -14,16 +13,16 @@ namespace nn {
 // Sequential.
 class FunctionalImpl : public torch::nn::Cloneable<FunctionalImpl> {
  public:
-  using Function = std::function<std::vector<Tensor>(std::vector<Tensor>)>;
-
-  explicit FunctionalImpl(Function function);
   explicit FunctionalImpl(std::function<Tensor(Tensor)> function);
```

Comment:
Maybe for a later PR:

Possible to templatize this, so you can write `Functional(at::max_pool2d, {2, 2})` or something? Basically, it would replace that call with `at::max_pool2d(x, {2, 2})`.